src/CodeSlideTitle.js

```diff
@@ -14,9 +15,13 @@ const styles = {
 };
 
 class CodeSlideTitle extends React.Component {
+  static propTypes = {
+      style: PropTypes.object
+  }
+
   render() {
     return (
-      <h1 style={styles}>{this.props.children}</h1>
+      <h1 style={{ ...styles, ...this.props.style }}>{this.props.children}</h1>
```

Comment:
I don't think you would need the spread operator here, as react already supports an array in `style` property.

I think `style={[styles, this.props.style]}` would do the job (and not be always creating a new object in render)